buildkite/src/Jobs/Lint/Fast.dhall

```diff
@@ -18,7 +18,6 @@ let commands =
       [ Cmd.run "./scripts/lint_codeowners.sh"
       , Cmd.run "./scripts/lint_rfcs.sh"
       , Cmd.run "make check-snarky-submodule"
-      , Cmd.run "./scripts/lint_preprocessor_deps.sh"
```

Comment:
The only library which uses `ppx_optcomp` is now `mina_node_config`. It turns out that this script doesn't work when there is only one lib to check, and it also doesn't seem relevant anymore.